torch_geometric/nn/conv/disen_conv.py

```diff
@@ -0,0 +1,155 @@
+from typing import List, Optional, Tuple, Union
+
+import torch
+import torch.nn.functional as F
+from torch import Tensor
+from torch_sparse import SparseTensor, matmul
+
+from torch_geometric.nn.conv import MessagePassing
+from torch_geometric.nn.dense.linear import Linear
+
+
+class DisenConv(MessagePassing):
+    r"""The graph convolutional operator from the `"Disentangled graph
+    convolutional networks"
+    <http://proceedings.mlr.press/v97/ma19a.html>`_ paper
+
+    Args:
+      in_channels (int): The number of input channels.
+      out_channels (int): The number of channels of the output feature.
+      K (int): The number of channels in the output.
+      T (int): The number of iterations. Defaults to 5
+      tao (float): The time decay parameter.
+      separate_channels (List[int]): This is a list of integers, which
+                specifies the number of channels for each factor.
+      weight_initializer (Optional[str]): The initializer for
```

Comment:
Is support for `weight_initializer` and `bias_initializer` desperately needed here? I think we should drop this to align better with existing GNN layers.

Reply:
I will remove them.